### PR TITLE
Setup prometheus in the system

### LINKED ISSUE #275: Setup prometheus in the system
### What is the expected behavior?

Setup prometheus deployment for helm chart.
In addition in would be good to have prometheus in docker-compose for local development too.

Epic https://github.com/Qiskit-Extensions/quantum-serverless/issues/139

### PR DESCRIPTION
**Pull request by psschwei**
Fixes #275 

Adds prometheus to helm install and docker compose files.
Also drops one more manager piece that was missed in previous PRs.

### CODE REVIEW

**Review by Tansito — CHANGES_REQUESTED** [review]

File: infrastructure/helm/quantumserverless/values.yaml [hunk 1]
```diff
@@ -91,15 +91,15 @@ ray-cluster:
           imagePullPolicy: IfNotPresent
           args:
           - --client-secret=CLIENTSECRET-CHANGEME
-          - --oidc-issuer-url=http://LOCAL-IP:31059/realms/quantumserverless
+          - --oidc-issuer-url=http://172.18.0.2:31059/realms/quantumserverless
```
[the remark this answers is not in the document]
Comment (psschwei) [reply infrastructure/helm/quantumserverless/values.yaml:94]: whoops, meant to stash that :facepalm:

File: infrastructure/helm/quantumserverless/values.yaml [hunk 2]
```diff
@@ -91,15 +91,15 @@ ray-cluster:
...
           - --oidc-extra-audience="account"
           - --email-domain="*"
           - --insecure-oidc-allow-unverified-email=true
           - --http-address=0.0.0.0:4180
           - --cookie-secret=SECRET0123456789
           - --provider=keycloak-oidc
           - --client-id=rayclient
-          - --upstream="http://HELM-RELEASE-kuberay-head-svc:8265"
+          - --upstream="http://quantum-serverless-kuberay-head-svc:8265"
```
[the remark this answers is not in the document]
Comment (psschwei) [reply infrastructure/helm/quantumserverless/values.yaml:102]: same as above :facepalm:

File: infrastructure/helm/quantumserverless/values.yaml [hunk 3]
```diff
@@ -256,4 +256,1134 @@ keycloak:
...
-  extraStartupArgs: "--import-realm"
\ No newline at end of file
+  extraStartupArgs: "--import-realm"
+
+# ===================
+# Prometheus 
+# ===================
+
+prometheusEnable: true
+rbac:
```
[the remark this answers is not in the document]
Comment (psschwei) [reply infrastructure/helm/quantumserverless/values.yaml]: :+1: 
helm didn't like an empty section, so I commented out the `prometheus:` line also

**Comment by psschwei** [comment]
@Tansito this one's ready for another look

**Review by Tansito — APPROVED** [review]
Thank @psschwei ! For me it's nice 😄

**Pull request merged**